docs/sources/k6/next/javascript-api/k6-browser/request/_index.md

```diff
@@ -12,7 +12,7 @@ The request that the browser performs can be retrieved from the [Response](https
 
 ## Supported APIs
 
-| Method                                                                                                                            | Playwright Relevant Distinctions                                                                                     |
+| Method                                                                                                                            | Relevant Distinctions                                                                                     |
```

Comment:
@ankur22 this is the other mention of Playwright on a table. Do you think we should still keep this? 🤔